clang/test/Driver/sycl-post-link-options-win.cpp

```diff
@@ -4,13 +4,26 @@
 // RUN:          -Xdevice-post-link -O0 %s 2>&1 \
 // RUN:   | FileCheck -check-prefix OPTIONS_POSTLINK_JIT_OLD %s
 // OPTIONS_POSTLINK_JIT_OLD: sycl-post-link{{.*}} "-O2" "-device-globals" "-properties" "-spec-const=native" "-split=auto" "-emit-only-kernels-as-entry-points" "-emit-param-info" "-symbols" "-emit-exported-symbols" "-emit-imported-symbols" "-split-esimd" "-lower-esimd" "-O0"
-
+// -------
+// Generate .o file as linker wrapper input.
+//
 // RUN: %clang -cc1 %s -triple x86_64-pc-windows-msvc -emit-obj -o %t.elf.o
-// RUN: clang-offload-packager -o %t.out --image=file=%t.elf.o,kind=sycl,triple=spir64
+// RUN: clang-offload-packager -o %t.out --image=file=%t.elf.o,kind=sycl,triple=spir64-unknown-unknown
 // RUN: %clang -cc1 %s -triple x86_64-pc-windows-msvc -emit-obj -o %t.o \
 // RUN:   -fembed-offload-object=%t.out
+//
+// Generate .o file as SYCL device library file.
+//
+// RUN: echo '' > %t.devicelib.cpp
+// RUN: %clang -cc1 %t.devicelib.cpp -triple spir64-unknown-unknown -aux-triple x86_64-pc-windows-msvc -fsycl-is-device -emit-llvm-bc -o %t.devicelib.bc
+// RUN: clang-offload-packager -o %t.devicelib.out --image=file=%t.devicelib.bc,kind=sycl,triple=spir64-unknown-unknown
+// RUN: %clang -cc1 %s -triple x86_64-pc-windows-msvc -emit-obj -o %t.devicelib.o \
+// RUN:   -fembed-offload-object=%t.devicelib.out
+//
+// Run clang-linker-wrapper test
+//
 // RUN: clang-linker-wrapper --dry-run --host-triple=x86_64-pc-windows-msvc \
-// RUN:   -sycl-device-library-location=%S/Inputs -sycl-device-libraries=libsycl-crt.new.obj \
+// RUN:   -sycl-device-library-location= -sycl-device-libraries=%t.devicelib.o \
```

Comment:
```suggestion
// RUN:   -sycl-device-libraries=%t.devicelib.o \
```
Do we need setting `-sycl-device-library-location` option?

Reply:
I had a check in my linker-wrapper code to look for empty library location and throw an error if it is not set. I think I do  not need to do that. Removed it.

Thanks